pkg/kmssigner/kmssigner.go

```diff
@@ -115,3 +130,30 @@ func (s *Signer) Sign(msg []byte) ([]byte, error) {
 
 	return resp.GetSignature(), nil
 }
+
+// VerifierKeyString returns a string which can be used to create a note
+// verifier based on a GCP KMS
+// [Ed25519](https://pkg.go.dev/golang.org/x/mod/sumdb/note#hdr-Generating_Keys)
+// key.
+func VerifierKeyString(ctx context.Context, c *kms.KeyManagementClient, kmsKeyName, noteKeyName string) (string, error) {
+	req := &kmspb.GetPublicKeyRequest{
+		Name: kmsKeyName,
+	}
+	resp, err := c.GetPublicKey(ctx, req)
+	if err != nil {
+		return "", err
+	}
+
+	publicKey, err := publicKeyFromPEM([]byte(resp.Pem))
+	if err != nil {
+		return "", err
+	}
+
+	h, err := keyHash(noteKeyName, publicKey)
+	if err != nil {
+		return "", err
+	}
+
+	prefixedPublicKey := append([]byte{algEd25519}, publicKey...)
+	return fmt.Sprintf("%s+%08x+%s", noteKeyName, h, base64.StdEncoding.EncodeToString(prefixedPublicKey)), nil
```

Comment:
I think you might be able to use `note.NewEd25519Verifier(noteKeyName, publicKey)` here?

Reply:
Thanks! Forgot we talked about this.